projects/plugins/jetpack/_inc/client/writing/custom-css.jsx

```diff
@@ -51,7 +51,7 @@ function CustomCss( props ) {
 				<div className="jp-custom-css-site-editor__text">
 					{ createInterpolateElement(
 						__(
-							'Hurray! Your theme supports site editing with blocks. <a>Tell me more.</a>',
+							'Your site has a block theme that allows you to apply custom CSS from the Site Editor. <a>Learn more.</a>',
```

Comment:
What do you think about linking to https://wordpress.org/documentation/article/styles-overview/#applying-custom-css instead?

Reply:
I like the idea! Done in 718870107629423b9f452036f91d569cfe31fcaa